sdk/messaging/azservicebus/internal/errors.go

```diff
@@ -175,7 +175,13 @@ func GetRecoveryKind(err error) recoveryKind {
 		}
 
 		// simple timeouts
```

Comment:
Should we have verbose logging for the 500?

Reply:
It happens naturally as part of current logging with the Retry() code, along with any other error.